Tests/test_SearchIO_model.py

```diff
@@ -163,8 +163,8 @@ def test_len(self):
         # len() should return the number of hits contained
         self.assertEqual(3, len(self.qresult))
 
-    def test_nonzero(self):
-        """Test QueryResult.__nonzero__."""
+    def test_bool(self):
+        """Test QueryResult.__bool__."""
         # nonzero should return true only if the qresult has hits
```

Comment:
The comment looks suitable for an update as well, considering the function is not called `test_nonzero` anymore.

Also, should [a similar test function for the Hit object](https://github.com/peterjc/biopython/blob/drop_cmp/Tests/test_SearchIO_model.py#L814) also be renamed, if this one is?

Reply:
Good catches, I'll update this - thanks!